### DISCUSSION TITLE
What is `UnoTextControlElevationBorderOverlayBrush`?

### DISCUSSION THREAD

**Discussion by xperiandri** [unoplatform/uno#17457] [Q&A] [answered]
When I copy Uno XAML with `UnoTextControlElevationBorderOverlayBrush` it cannot find that

**Comment by morning4coffe-dev** [comment]
Hey @xperiandri, where did you copy this Resource from? I think it was used in the past with [Uno_themeresources](https://github.com/MartinZikmund/Uno/blob/75686cea4da809e04c3b1c1317ef7f13fc4fd17d/src/Uno.UI.FluentTheme.v2/Resources/Version2/Priority01/Uno_themeresources.xaml#L11).

**Reply by MartinZikmund** [reply, marked as answer]
Yes it is a legacy brush to work around an issue with `LinearGradientBorder` rendering, but it is no longer needed. It should no longer appear in any of the up-to-date styles
